pkg/controllers/cspi-controller/handler.go

```diff
@@ -185,6 +186,16 @@ func (c *CStorPoolInstanceController) reconcile(key string) error {
 	common.SyncResources.Mux.Unlock()
 	// This case is possible incase of ephemeral disks
 	if !cspi.IsEmptyStatus() && !cspi.IsPendingStatus() {
+		// This scenario will occur when the zpool command hung due to same/someother
+		// bad disk existence in the system (or) when the underlying pool disk is lost.
+
+		// NOTE: If zpool command hung cstor-pool container in pool manager will restart
+		// because of due liveness on the pool. If cstor-pool container is killed then
+		// zpool commands will error out and fall into this scenario
+		c.recorder.Event(cspi, corev1.EventTypeWarning,
+			string(common.FailedSynced),
+			fmt.Sprintf("Unable to import a pool may be due to underlying pool disk might be lost or bad disk might be attached to the node"),
```

Comment:
@sonasingh46 is there any better suggestion on event message?

Reply:
`Failed to import the pool as the underlying pool might be lost or the disk has gone bad`